src/resources/users/user-model.js

```diff
@@ -64,9 +64,9 @@ async function add(user) {
   return findById(id);
 }
 
-async function update(id, body) {
+async function update(email, body) {
   const updatedUser = await db('users')
-    .where({ id })
```

Comment:
You don't can change models because we are using them already on the frontend. You need to create a new model if you dont can use this one